cruise-control/src/yaml/endpoints/addBroker.yaml

```diff
@@ -0,0 +1,184 @@
+        schema:
+          type: boolean
+          default: false
+      - name: verbose
+        in: query
+        description: Return detailed state information.
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_demoted_brokers
+        in: query
+        description: Whether to allow leader replicas to be moved to recently demoted broker.
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_removed_brokers
+        in: query
+        description: Whether to allow replicas to be moved to recently removed broker.
+        schema:
+          type: boolean
+          default: false
+      - name: replica_movement_strategies
+        in: query
+        description: Replica movement strategies to use.
+        schema:
+          type: array
+          items:
+            type: string
+          default: null
+          example: ["PrioritizeLargeReplicaMovementStrategy", "PostponeUrpReplicaMovementStrategy"]
+      - name: execution_progress_check_interval_ms
+        in: query
+        description: Execution progress check interval in milliseconds.
+        schema:
+          type: integer
+          format: int64
+          minimum: 5000
+      - name: stop_ongoing_execution
+        in: query
+        description: Whether to stop the ongoing execution (if any) and start executing the given request.
+        schema:
+          type: boolean
+          default: false
+      - name: reason
+        in: query
+        description: Reason for request.
+        schema:
+          type: string
+      - name: review_id
+        in: query
+        description: Review id for 2FA.
+        schema:
+          type: integer
+          format: int32
+          minimum: 0
+      - name: data_from
+        in: query
+        description: Whether to calculate proposal from available valid partitions or valid windows.
+        schema:
+          type: string
+          default: VALID_WINDOWS
+      - name: replication_throttle
+        in: query
+        description: Upper bound on the bandwidth in bytes per second used to move replicas.
+        schema:
+          type: integer
+          format: int64
+          minimum: 1
+    responses:
+      '200':
+        description: |-
+          Successful add brokers response
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/optimizationResult.yaml#/OptimizationResult'
+          text/plain:
+             schema:
+               type: string
+      '202':
+        description: |-
+          Add brokers in progress
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/progressResult.yaml#/ProgressResult'
+          text/plain:
+             schema:
+               type: string
+      # Response for all errors
+      default:
+        description: |-
+          Unexpected error
```

Comment:
(Also apply to other yaml files added in this PR)`Unexpected error` -> `Error response.`(to be consistent with existing endpoints)